src/io/Startup.ts

```diff
@@ -24,13 +24,13 @@ export default async function(lancerVer: string, ccVer: string, store: any): Pro
 
   await Auth.currentAuthenticatedUser()
     .then(user => {
-      userstore.setAws(user, 'appLoad')
+      userstore.setAws({user: user, condition: 'appLoad', noSync: noSync})
     })
     .catch(() => {
       userstore.loadUser()
     })
 
-  await dataStore.loadExtraContent()
+  await dataStore.refreshExtraContent()
```

Comment:
Refresh the available LCPs (which still involves loading those LCPs) to prevent loading duplicate LCPs into the Content Manager panel.